oat-sa/extension-tao-delivery
# add proctorio placeholder element

Ticket: https://oat-sa.atlassian.net/browse/NEX-907

I have added a min-height 300px to `body` tag which solves the issue.

I believe the real issue lies in Proctoio’s logic of giving height to their iFrame and not in tao but since we can’t really modify the way they render tao in iFrame we ll have to fix it in tao. Our test-runner-scope element relies on `viewheight` to set its height and Proctorio has `viewheight` of 0 (of their iframe) so that's why the test runner is not visible.

**How to test?**

- Have a working remote proctoring environment
- Create a delivery
- Execute the delivery with a test taker (not as a guest)
- Follow the on-screen instructions of Proctorio
- You should be able to launch tao test runner and take the test without any issues
